### ISSUE TITLE
Improve about window focus

### ISSUE THREAD

**Issue by pacocoursey** [wulkano/kap#524]
**macOS Version:** 10.13.6
**Kap Version:** 3.0.0-beta.1.1006

#### Steps to reproduce
- Open Kap (default recording interface)
- Click the ... icon and then "About Kap Beta"

#### Current behaviour
- The About window is opened
- It is not focused or interact-able because the recording interface takes mouse focus
- Cannot close or interact with the About window until you exit the recording interface

#### Expected behaviour
- Exit the recording interface
- Open the About window
- Basically just use the same behaviour as the Preferences window

#### Screenshot
<p align="center">
<img width="332" alt="screen shot 2018-09-24 at 11 36 20 am" src="https://user-images.githubusercontent.com/34928425/45968413-1d548480-bfee-11e8-8644-c52e36352116.png">
</p>

It feels natural that a newly created window should receive focus and is not hidden behind the recording interface.
